test/pageobjects/Camera.js

```diff
@@ -39,6 +43,17 @@ class Camera extends BasePage {
     this.shutterButton().click()
   }
 
+  async enableCameraAccessIfNecessary() {
+    if (browserName === 'safari' || browserName === 'Safari') {
+      if (
+        this.enableCameraButton().isDisplayed() === true &&
+        this.allowCameraPermissionImage().isDisplayed() === true
+      ) {
```

Comment:
Should be enough to just write it like this 
```suggestion
      if (
        this.enableCameraButton().isDisplayed() &&
        this.allowCameraPermissionImage().isDisplayed()
      ) {
```
shorter and also makes it more readable IMO.